catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/gmti/NitfGmtiTransformer.java

```diff
@@ -135,6 +137,47 @@ private String setLocation(Metacard metacard) {
         return null;
     }
 
+    private void transformAircraftLocation(Metacard metacard) {
+
+        String aircraftLocation = formatAircraftLocation(metacard);
+
+        try {
+            LOGGER.debug("Formatted Aircraft Location = " + aircraftLocation);
+
+            if (aircraftLocation != null) {
+                // validate the wkt
+                WKTReader wktReader = new WKTReader(geometryFactory);
+                Geometry geometry = wktReader.read(aircraftLocation);
+
+                LOGGER.debug("Setting the metacard attribute [{}, {}]", Isr.DWELL_LOCATION,
+                        aircraftLocation);
+                metacard.setAttribute(new AttributeImpl(Isr.DWELL_LOCATION, aircraftLocation));
+            }
+
+        } catch (ParseException e) {
+            LOGGER.warn(e.getMessage(), e);
+        }
+    }
+
+    private String formatAircraftLocation(Metacard metacard) {
+        Attribute aircraftLocation = metacard
+                .getAttribute(MtirpbAttribute.AIRCRAFT_LOCATION.getAttributeDescriptor().getName());
+
+        if (aircraftLocation != null
+                && StringUtils.isNotEmpty(aircraftLocation.getValue().toString())) {
+
+            String unformattedAircraftLocation = aircraftLocation.getValue().toString();
+
+            StringBuilder sb = new StringBuilder("POINT (");
+            parseLocation(sb, unformattedAircraftLocation);
```

Comment:
can we depend on the coordinate order being LONG LAT in all cases?  Is this field always decimal degrees?